dist/reporttable.js

```diff
@@ -14,155 +14,239 @@
  *
  *
  */
- 
+
 (function ($) {
 
     var ReportTable = function(container, options)
     {
         this.container = $(container);
-        this.options = $.extend(true, {}, this.defaultOptions, options);    
-        
+        this.options = $.extend(true, {}, this.defaultOptions, options);
+
         this.buildTable();
     }
-    
-    
+
     //this function will rebuild the table to be converted into a with static columns and a fixed header
     ReportTable.prototype.buildTable = function()
     {
         this.reset();
-        
+
         this.refreshFunct = $.proxy(this.scroll, this);
         this.resizeFunct = $.proxy(this.resize, this);
-        
-        $(window).scroll(this.refreshFunct);        
+
+        $(window).scroll(this.refreshFunct);
         $(window).resize(this.resizeFunct);
-    
+
         var scrollEventObject;
+        var that = this;
         if (scrollEventObject = this.options.scrollEventObject)  {
-            var that = this;
             scrollEventObject.scroll(function() {that.scroll()});
-        } else { 
+        } else {
             this.interval = setInterval($.proxy(this.scroll, this), this.options.pollInterval);
         }
         this.resizeInterval = setInterval($.proxy(this.resize, this), this.options.resizePollInterval);
-        this.container.bind("remove", $.proxy(function(){ 
+        this.container.bind("remove", $.proxy(function(){
             clearInterval(this.interval);
             clearInterval(this.resizeInterval);
             $(window).unbind('scroll', this.refreshFunct);
             $(window).unbind('resize', this.resizeFunct);
             this.scroller.remove();
-        }, this));      
+        }, this));
+
+        this.frozenIndices = [];
+        this.container.on("colFreezeToggle", function (event, index) {
+            var widths = that.getOuterWidthsFromFirstRow();
+            var headerHeight = that.getHeightFromHeader();
+            var rowHeight = that.getHeightFromFirstRow();
+            var frozenIndex = that.frozenIndices.indexOf(index);
+
+            frozenIndex == -1 ? 
+                that.frozenIndices.push(index) : that.frozenIndices.splice(frozenIndex, 1);
+
+            $(".reporttable_header, .koddiTableGraph, .reporttable_footer").find("tr").each(function () {
```

Comment:
Also, `.koddiTableGraph` is not part of this repo and is specific to the original table object, so we should change that selector. Maybe to select the second table of the container?